src/main/resources/view/BrownTheme.css

```diff
@@ -1,3 +1,11 @@
+.root {
```

Comment:
I really like these colours and I am sure it will help to make our app more visually appealing!